### PR TITLE
v1.9: docs: Fix cilium-runtime image bump instructions

### PR DESCRIPTION
**Pull request by joestringer**
This is a process that reliably works, unlike the previous one which
relies on Quay.io's ability to pull images from DockerHub. It does
require a core maintainer to prepare the changes, but that's fine for
now since v1.9 should change infrequently now.

### CODE REVIEW

File: Documentation/contributing/development/images.rst [hunk 1]
```diff
@@ -124,95 +124,72 @@ Image dependency:
...
-   have a branch that contains changes, select the branch that contains the new
-   changes.
+#. Log into skopeo into both dockerhub and quay.io, and copy the image across:
 
-.. image:: ../../images/cilium-quayio-tag-4.png
-    :align: center
+   .. code-block:: shell-session
 
-8. After selecting your branch click on "Start Build".
+      $ skopeo login docker.io
```
Comment (aditighag) [inline Documentation/contributing/development/images.rst:152]: Does it ask prompt for a username?

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
cc @tklauser FYI this will only be for the v1.9 branch.

**Review by tklauser — APPROVED** [review]

**Comment by aanm** [comment]
Only doc changes no need to re-run the CI.

File: Documentation/contributing/development/images.rst (hunk 1 above)
Comment (joestringer) [reply Documentation/contributing/development/images.rst:152]: I think so, though I don't know what kind of configuration I may have prepared locally to make this work. I'd imagine that if someone other than Andre / I need to follow these instructions, then hopefully this step will at least ensure that they figure out how to set up skopeo correctly.

That said, this is only applicable for v1.9 as the newer branches have better infrastructure in the tree for handling these updates so we don't need to rely on developers having direct access to the upstream container repositories.

**Pull request merged**